src/ui/marker.js

```diff
@@ -289,6 +289,14 @@ export default class Marker extends Evented {
             }
             this._popup = popup;
             if (this._lngLat) this._popup.setLngLat(this._lngLat);
+
+            this._element.addEventListener('keypress', ((e) => {
+                const code = e.charCode || e.keyCode;
+
+                if ((code === 32) || (code === 13)) { // space or enter
+                    this.togglePopup();
+                }
+            }).bind(this));
```

Comment:
No need to `bind` because this is an arrow function.

Reply:
Thanks! I originally used a regular function and needed bind, but swapped to the arrow function when I was reminded of the style guide by lint, good to know arrow functions don't need to bind this.